### PR TITLE
For ewels

### PR DESCRIPTION
**Pull request by aledj2**

### CODE REVIEW

**Comment by aledj2** [comment]
fixes #661 and partially addresses that raised in #663

File: multiqc/modules/peddy/peddy.py [hunk 1]
```diff
@@ -121,11 +122,16 @@ def parse_peddy_csv(self, f, pattern):
                 parsed_data[s_name] = dict()
                 for i, v in enumerate(s):
                     if i not in s_name_idx:
+                        if headers[i] == "error" and pattern == "sex_check":
+                            if v == "True":
```
Comment (ewels) [inline multiqc/modules/peddy/peddy.py]: Can compress this a bit I think:
```python
v = "True" if v == "False" else "False"
```

File: multiqc/modules/peddy/peddy.py [hunk 2]
```diff
@@ -74,10 +74,11 @@ def __init__(self):
 
         # Relatedness plot
         self.peddy_relatedness_plot()
-
+
```
Comment (ewels) [inline multiqc/modules/peddy/peddy.py]: Would be nice if you could clean up these whitespace changes in the diff - they all get highlighted in a bright orange in my editor so it saves me removing them later 😉

**Comment by aledj2** [comment]
verifybamid.py has a comma removed at the end of a dicitonary. I think that's correct - I don't get any errors but let me know if it's wrong

**Comment by ewels** [comment]
Perfect, thanks! A comma on the final element of a Python iterable (dictionary, list etc) is not required, but it doesn't break if it's there either.

**Comment by ewels** [comment]
Note that you may get some git nastyness when you pull changes after this squash merge. You'll probably need to rebase your git history to fix this. Worth doing straight away before you start making any future code changes :)

Phil

**Comment by ewels** [comment]
ps. To avoid me doing squash merges in the future, you can use interactive rebasing to reduce the number of commits - there were 39 commits here for quite a minor change which is why I squashed them. See for example https://github.com/mockito/mockito/wiki/Using-git-to-prepare-your-PR-to-have-a-clean-history

**Pull request merged**